pkg/api/server.go

```diff
@@ -142,8 +142,12 @@ func (s *Server) startGRPC() error {
 		grpc.Creds(insecure.NewCredentials()),
 		grpc.UnaryInterceptor(middleware.ChainUnaryServer(unary...)),
 		grpc.StreamInterceptor(middleware.ChainStreamServer(stream...)),
+		grpc.KeepaliveParams(keepalive.ServerParameters{
+			Time: 5 * time.Minute,
+		}),
 		grpc.KeepaliveEnforcementPolicy(keepalive.EnforcementPolicy{
-			MinTime: 15 * time.Second,
+			PermitWithoutStream: true,
+			MinTime:             15 * time.Second,
```

Comment:
Also, @nplasterer. I just added this config setting in a previous PR to turn it to 15 seconds. Before it was set to 5 minutes. I wonder how many Rust streaming problems were related to that.

Reply:
O wow! 🤞